### PR TITLE
Add links to import and publish Learn tutorial

### PR DESCRIPTION
**Pull request by im2nguyen**
## Labels

- [ ] inaccuracy
- [x] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout

### CODE REVIEW

File: content/source/docs/registry/providers/publishing.html.md [hunk 1]
```diff
@@ -12,13 +12,16 @@ Anyone can publish and share a provider by signing into the Registry using their
 
 This page describes how to prepare a [Terraform provider](/docs/language/providers/index.html) for publishing, and how to publish a prepared provider using the Registry's interface.
 
+> **Hands-on:** Try the [Release and Publish a Provider to the Terraform Registry](https://learn.hashicorp.com/tutorials/terraform/provider-release-publish?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn. In this tutorial, you will create a release for your custom Terraform provider and prepare your provider to publish to the Terraform Registry.
```
Comment (topfunky) [inline content/source/docs/registry/providers/publishing.html.md:15]:
```suggestion
-> **Hands-on:** Try the [Release and Publish a Provider to the Terraform Registry](https://learn.hashicorp.com/tutorials/terraform/provider-release-publish?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn. In this tutorial, you will create a release for your custom Terraform provider and prepare your provider to publish to the Terraform Registry.
```

Should this be a callout or do we have a history of only using a blockquote? It feels like blockquote should be used for actual quotes, not announcements or related content.

File: content/source/docs/registry/providers/publishing.html.md [hunk 2]
```diff
@@ -12,13 +12,16 @@ Anyone can publish and share a provider by signing into the Registry using their
...
+
 ## Preparing your Provider
 
 ### Writing a Provider
 
 Providers published to the Terraform Registry are written and built in the same way as other Terraform providers. A variety of resources are available to help our contributors build a quality integration:
 
-- The [Call APIs with Terraform Providers](https://learn.hashicorp.com/collections/terraform/providers?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn
+> **Hands-on:** Try the [Call APIs with Terraform Providers](https://learn.hashicorp.com/collections/terraform/providers?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn. In these tutorials, you will create a custom Terraform provider, complete with authentication and create/read/update/delete/import functionality.
```
Comment (topfunky) [inline content/source/docs/registry/providers/publishing.html.md:23]:
```suggestion
-> **Hands-on:** Try the [Call APIs with Terraform Providers](https://learn.hashicorp.com/collections/terraform/providers?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn. In these tutorials, you will create a custom Terraform provider, complete with authentication and create/read/update/delete/import functionality.
```

File: content/source/docs/extend/resources/import.html.md [hunk 3]
```diff
@@ -22,6 +22,8 @@ $ terraform import example_thing.foo abc123
 
 Implementing import support requires three changes: an `Importer` `State` function in the resource code, a `TestStep` with `ImportState: true` in the acceptance tests, and documentation of the import ID format.
 
+> **Hands-on:** Try the [Implement Import](https://learn.hashicorp.com/tutorials/terraform/provider-import?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn. In this tutorial, you will implement the import functionality on an example Terraform provider.
```
[the remark this answers is not in the document]
Comment (judithpatudith) [reply content/source/docs/extend/resources/import.html.md:25]: It should be a block quote :)

[the remark this answers is not in the document]
Comment (im2nguyen) [reply content/source/docs/extend/resources/import.html.md:25]: Yep, all the other docs callout to Learn use `>`

image

**Pull request merged**